coalib/bearlib/abstractions/ExternalBearWrap.py

```diff
@@ -6,9 +6,9 @@
 from coalib.bears.LocalBear import LocalBear
 from coala_utils.decorators import enforce_signature
 from coalib.misc.Shell import run_shell_command
-from coalib.results.Diff import Diff
 from coalib.results.Result import Result
 from coalib.results.SourceRange import SourceRange
+# Ignore PyUnusedCodeBear
```

Comment:
`from coalib.results.SourceRange import SourceRange # Ignore PyUnusedCodeBear` would be on line 10 right? thats 77 chars.